build/dnvm.ps1

```diff
@@ -84,7 +84,7 @@ Set-Variable -Option Constant "CommandFriendlyName" ".NET Version Manager"
 Set-Variable -Option Constant "DefaultUserDirectoryName" ".dnx"
 Set-Variable -Option Constant "OldUserDirectoryNames" @(".kre", ".k")
 Set-Variable -Option Constant "RuntimePackageName" "dnx"
-Set-Variable -Option Constant "DefaultFeed" "https://www.nuget.org/api/v2"
+Set-Variable -Option Constant "DefaultFeed" "https://www.myget.org/F/aspnetvnext/api/v2"
```

Comment:
Late comment on this: dnvm shouldn't be hardcoded to 2 feeds. We have at least 3 (NuGet, myget/vnext, myget/release). We need to find a better solution. Maybe dnvm should only have one configurable feed and not hardcode multiple options. Today, this breaks the release build on non CI machines because it brings beta7 dnx when build beta6 release